java/src/com/ibm/streamsx/topology/TStream.java

```diff
@@ -89,6 +90,41 @@
      * @return Filtered stream
      */
     TStream<T> filter(Predicate<T> filter);
+    
+    /**
+     * Split a stream into {@code n} streams as specified by {@code splitter}.
+     * 
+     * <P>
+     * For each tuple on the stream, {@code splitter.applyAsInt(tuple)} is called.
+     * The return value {@code r} determines the destination stream:
+     * <pre>
+     * if r < 0 the tuple is discarded
+     * else it is sent to the stream at position (r % n) in the returned array.
+     * </pre>
+     * </P>
+     * 
+     * <P>
+     * Example of round robin splitting a stream of {@code String} into 5 channels.
```

Comment:
Since split is a logical split, can we have suitable example, e.g. based upon a type of field in the object. Also if a comment could be added as to what makes it different to parallel().